webapp/src/main/java/com/box/l10n/mojito/service/branch/notification/phabricator/BranchNotificationMessageBuilderPhabricator.java

```diff
@@ -21,9 +22,25 @@
     @Autowired
     BranchUrlBuilder branchUrlBuilder;
 
+    @Value("${l10n.phabricator.notification.message.new:We received your strings! " +
+            "Please **add screenshots** as soon as possible and **wait for translations** before releasing. }")
+    String newNotificationMsg;
+
+    @Value("${l10n.phabricator.notification.message.updated:Your branch was updated with new strings! " +
+            "Please **add screenshots** as soon as possible and **wait for translations** before releasing. }")
+    String updatedNotificationMsg;
+
+    @Value("{l10n.phabricator.notification.message.noMoreStrings:The branch was updated and there are no more strings to translate.}")
+    String noMoreStrings;
+
+    @Value("{l10n.phabricator.notification.message.noMoreStrings:Translations are ready!!}")
```

Comment:
message.noMoreStrings -> message.translationsReadyMsg? 

you want to update the test to catch that too. Looks like only one of the tests was updated with all strings